README.md

```diff
@@ -58,6 +58,8 @@ COMMANDS
   preview  create a documentation preview from the given file or URL
 ```
 
+Please check [Bump help center](https://help.bump.sh/continuous-integration) for more Continuous Integration usage details.
```

Comment:
I think we should link to the dedacted CLI help page instead of the Continuous Integration page:

```suggestion
Please check the [Bump CLI help page](https://help.bump.sh/bump-cli) for more CLI usage details.
```

Reply:
Indeed, much better 👍